cmd/release/utils/values/searchprs.go

```diff
@@ -12,34 +12,53 @@ const (
 	baseQuery = "repo:aws/eks-distro is:pr is:merged"
 )
 
-func GetChangelogPRs(releaseVersion string) (string, error) {
+func GetChangelogPRs(releaseVersion string, overrideNumber int) (string, error) {
 	githubClient := github.NewClient(nil)
 
 	ctx := context.Background()
-	opts := &github.SearchOptions{}
+	opts := &github.SearchOptions{Sort: "updated"}
 	//Get the date of the last document release for the release version
-	prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged label:release label:documentation repo:aws/eks-distro label:" + releaseVersion, opts)
+	prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged label:release label:documentation repo:aws/eks-distro label:"+releaseVersion, opts)
 	if err != nil {
 		return "", fmt.Errorf("getting PRs from %v: %w", githubClient, err)
 	}
 
-	//Select the most recent pr from the above query and format the date expected for the go-github client
-	lastDocRelease := prs.Issues[0].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+	lastDocRelease := "2006-01-02T15:04:05+00:00"
+	prevDocRelease := "2006-01-02T15:04:05+00:00"
+	if len(prs.Issues) > 0 {
+		//Select the most recent pr from the above query and format the date expected for the go-github client
+		lastDocRelease = prs.Issues[0].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+	} else {
+		//With no document releases we need to be a little bit clever to generate unannounced changelogs.
+		//This finds the
+		opts = &github.SearchOptions{Sort: "updated", Order: "asc"}
+		prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged 'Bumped Production release number' in:title label:release label:"+releaseVersion, opts)
+		if err != nil {
+			return "", fmt.Errorf("get PRs from %v: %w", githubClient, err)
+		}
+		if overrideNumber == 1 {
+			lastDocRelease = prs.Issues[overrideNumber-1].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+		} else {
+			lastDocRelease = prs.Issues[overrideNumber-1].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+			prevDocRelease = prs.Issues[overrideNumber-2].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+		}
```

Comment:
Maybe this? Though double check that this is correct. 
```suggestion
		lastDocRelease = prs.Issues[overrideNumber-1].ClosedAt.Format("2006-01-02T15:04:05+00:00")
		if overrideNumber != 1 {
			prevDocRelease = prs.Issues[overrideNumber-2].ClosedAt.Format("2006-01-02T15:04:05+00:00")
		}
```